src/App.tsx

```diff
@@ -1,61 +1,115 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getUserId } from './services/User';
+
+const initialTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserId(todo.userId),
```

Comment:
The `getUserId` function is used here, but it seems to be intended to retrieve a user object rather than just an ID. Ensure that `getUserId` returns the correct user object or adjust the logic to retrieve the user object correctly.